Code/GraphMol/ChemReactions/ReactionWriter.cpp

```diff
@@ -113,7 +113,8 @@ void write_template(std::ostringstream &res, RDKit::ROMol &tpl) {
   }
   RDKit::FileParserUtils::moveAdditionalPropertiesToSGroups(trwmol);
 
-  res << RDKit::FileParserUtils::getV3000CTAB(trwmol, -1);
+  boost::dynamic_bitset<> wasAromatic(trwmol.getNumBonds());
+  res << RDKit::FileParserUtils::getV3000CTAB(trwmol, wasAromatic, -1);
```

Comment:
Due to the new getV3000CTab internally using the wasAromatic bitset, does this need to be changed or can it be the original:

```
res << RDKit::FileParserUtils::getV3000CTAB(trwmol, -1);
```

Reply:
you're right; there's no reason to change this.